liana-gui/src/app/view/home.rs

```diff
@@ -4,28 +4,63 @@ use std::{collections::HashMap, time::Duration, vec};
 use iced::{
     alignment,
     widget::{Container, Row, Space},
-    Alignment, Length,
+    Alignment::{self, Center},
+    Length,
 };
 
 use liana::miniscript::bitcoin;
 use liana_ui::{
     color,
     component::{amount::*, button, card, event, form, spinner, text::*},
-    icon, theme,
+    icon::{self, cross_icon},
+    theme,
     widget::*,
 };
 
 use crate::{
     app::{
         cache::Cache,
         error::Error,
-        menu::Menu,
+        menu::{self, Menu},
         view::{coins, dashboard, label, message::Message},
         wallet::SyncStatus,
     },
     daemon::model::{HistoryTransaction, Payment, PaymentKind, TransactionKind},
 };
 
+const RESCAN_WARNING: &str = "As this wallet was restored from a backup, you may need to rescan the blockchain to see past transactions.";
+
+fn rescan_warning<'a>() -> Element<'a, Message> {
+    Container::new(
+        Column::new()
+            .spacing(5)
```

Comment:
```suggestion
            .spacing(10)
```

I think it will look better

Reply:
Done.